app/controllers/admin/communities_controller.rb

```diff
@@ -285,7 +292,13 @@ def update_settings
 
     maybe_update_payment_settings(@current_community.id, params[:community][:automatic_confirmation_after_days])
 
-    MarketplaceService::API::Api.configurations.update(community_id: @current_community.id, main_search: params[:main_search]) if feature_enabled?(:location_search)
+    if(feature_enabled?(:location_search))
```

Comment:
There's a helper method related to this: https://github.com/sharetribe/sharetribe/blob/master/app/helpers/feature_flag_helper.rb#L15